localisation/00_wc_random_artifact_names.csv

```diff
@@ -92,6 +92,19 @@ artifact_legendary_origin_8;[GetGrammaticalNumberThisThese] [This.GetName] [GetG
 artifact_legendary_origin_9;[GetGrammaticalNumberThisThese] [This.GetName] holds the secret to existence for those who can decipher [GetGrammaticalNumberItsTheir] intricate design;ce [This.GetName] d�tient le secret de l'existence pour ceux qui peuvent d�chiffrer sa conception complexe;;;;;;;;;;;;x
 artifact_legendary_origin_10;materials from what [GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] crafted do not exist in this world;les mat�riaux � partir desquels ce [This.GetName] a �t� cr�� n'existent pas dans ce monde;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+artifact_looted_origin_1;Some individual was kind enough to "donate" [GetGrammaticalNumberThisThese] [This.GetName] to soldiers of [target_raiding_ruler.GetTitledFirstName] when they arrived in [target_looted_holding.GetName]. Locals did not hear about that person ever since.;;;;;;;;;;;;;x
```

Comment:
Changed the last sentence so that it matches with the tense.
```suggestion
artifact_looted_origin_1;Some individual was kind enough to "donate" [GetGrammaticalNumberThisThese] [This.GetName] to soldiers of [target_raiding_ruler.GetTitledFirstName] when they arrived in [target_looted_holding.GetName]. Locals have not heard about that person ever since.;;;;;;;;;;;;;x
```